dspace/config/registries/openaire4-types.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE dspace-dc-types SYSTEM "dspace-dc-types.dtd">
+
+<!--
+  This document was based on the OpenAIRE4 Guidelines for Literature Repositories
+  https://openaire-guidelines-for-literature-repository-managers.readthedocs.io/en/v4.0.0/index.html
+  -->
+  
+<dspace-dc-types>
+    <dspace-header>
+        <title>OpenAIRE4 fields definition</title>
+    </dspace-header>
+
+    <dc-schema>
+        <name>oaire</name>
+        <namespace>http://namespace.openaire.eu/schema/oaire/</namespace>
+    </dc-schema>
+	
+    <dc-type>
+        <schema>oaire</schema>
+        <element>fundingStream</element>
+        <scope_note>Name of the funding stream</scope_note>
+    </dc-type>
+	
+    <dc-type>
+        <schema>oaire</schema>
+        <element>awardNumber</element>
+        <scope_note>Project grantId or awardNumber</scope_note>
+    </dc-type>
+	
+    <dc-type>
+        <schema>oaire</schema>
+        <element>awardURI</element>
+        <scope_note>URI of the project landing page provided by the funder for more information about the award (grant).</scope_note>
+    </dc-type>
+
+    <dc-type>
+        <schema>oaire</schema>
+        <element>awardTitle</element>
+        <scope_note>Title of the project, award or grant.</scope_note>
+    </dc-type>
+	
+    <dc-type>
+        <schema>oaire</schema>
+        <element>version</element>
+        <scope_note>Use either a version number or the label of the vocabulary term as value.</scope_note>
+    </dc-type>
+	
+    <dc-type>
+        <schema>oaire</schema>
+        <element>citationTitle</element>
+        <scope_note>The title name of the container (e.g. journal, book, conference) this work is published in. This property is considered to be part of the bibliographic citation.</scope_note>
```

Comment:
I would prefer `oaire.citation.titel` (and likewise for the other elements `oaire.citation.volume`...). I know that the field is specified without separate element and qualifier in the OpenAIRE documentation, but I think it would be a big usability improvement. For example we could use a `qualdrop` field in the submission process and would not require separate fields for every element.

Reply:
I brought this to discuss with @josekarvalho and we think your suggestion doesn't change the essence of what OpenAIRE proposed and since the fields are specifically described we won't need the qualifiers. Despite we don't have any specific use-case for `qualdrop` within the citation fields, others might have. We will make the changes as you suggested.